qa/common/inferentia_perf_analyzer_input_data_json/simple-model.py

```diff
@@ -24,33 +24,104 @@
+    model_dir ='add_sub_model'
+    compiled_model_dir = name
+    shutil.rmtree(model_dir, ignore_errors=True)
+    shutil.rmtree(compiled_model_dir, ignore_errors=True)
+    if (tfVersion == 1):
+        with tf.Session() as sess:
+            # Export SavedModel
+            input0 = tf.placeholder(tf.int64,[None, 4], "INPUT__0")
+            input1 = tf.placeholder(tf.int64,[None, 4], "INPUT__1")    
+            output0 = tf.add(input0, input1, "OUTPUT__0")
+            output1 = tf.subtract(input0, input1, "OUTPUT__1")
+            tf.compat.v1.saved_model.simple_save(session=sess,
+                                                    export_dir=model_dir,
+                                                    inputs={
+                                                        "INPUT__0": input0,
+                                                        "INPUT__1": input1
+                                                    },
+                                                    outputs={
+                                                        "OUTPUT__0": output0,
+                                                        "OUTPUT__1": output1
+                                                    })
+        # Compile using Neuron
+        tfn.saved_model.compile(model_dir,
+                                compiled_model_dir,
+                                batch_size=batch_size,
+                                dynamic_batch_size=True)
+    elif (tfVersion == 2):
+        # TODO: ADD tests for tensorflow2
+        raise Exception("TensorFlow2 not yet supported")
+    else:
+        raise Exception("Unrecognized Tensorflow version: {}".format(tfVersion))
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--model_type',
+                        type=str,
+                        required=True,
+                        choices=['pytorch', 'tensorflow'],
+                        help='''The type of the compiled model. Currently,
+                        only supports \"pytorch\" and \"tensorflow\".''')
+    parser.add_argument('--name',
+                        type=str,
+                        required=True,
+                        help='The name of the compiled model')
+    parser.add_argument('--tf_version',
+                        type=int,
+                        choices=[1, 2],
+                        help='Version of tensorflow for compiled model')
+    parser.add_argument('--batch_size',
+                        type=int,
+                        default=1,
+                        help='The batch size for the compiled model')
+
+    FLAGS, unparsed = parser.parse_known_args()
+    if len(unparsed) > 0:
+        raise Exception("Unrecognized options: {}".format(unparsed))
+    if FLAGS.model_type == 'tensorflow':
+        import os
+        import shutil
+        import tensorflow as tf
+        import tensorflow.neuron as tfn
+        print("name is {}".format(FLAGS.name))

```

Comment:
Yes

Reply:
You can move it outtside the if block since it is printed in both cases. Not sure why you would print the name that the user is providing but I do not think it is unnecessary